### PR TITLE
#102 Use coveragepy for python code coverage whenever possible

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

File: util/envDynawo.sh [hunk 1]
```diff
@@ -965,7 +965,10 @@ build_tests_coverage() {
   ## for unit test, no need to generate modelica models
   build_dynawo_core || error_exit "Error during build_dynawo_core."
   build_dynawo_models_cpp || error_exit "Error during build_dynawo_models_cpp."
-
+  if ${DYNAWO_PYTHON_COMMAND} -c "import coverage" &> /dev/null; then
+    DYNAWO_PYTHON_COMMAND_SAVE=${DYNAWO_PYTHON_COMMAND}
+    export_var_env_force DYNAWO_PYTHON_COMMAND="coverage run"
```
Comment (gautierbureau) [inline util/envDynawo.sh]: Is there any additional package to install on fedora to be able to run this command ? If yes we need to add it in the requirements for developpers and in the commands we provide to install dynawo in the README

Comment (rosiereflo) [reply util/envDynawo.sh]: the command is "pip install coverage". However it is not mandatory for dynawo developers (unless they are working on coverage), it is mostly useful for sonar.

Comment (rosiereflo) [reply util/envDynawo.sh]: I already added this installation in the dockerfiles normally

Comment (gautierbureau) [reply util/envDynawo.sh]: ok

**Pull request merged**